tests/flask_app/test_routes.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+import io
+
+from flask_app.flask_main import create_app
+
+@pytest.fixture
+def client():
+   app = create_app()
+   app.config["TESTING"] = True
+   app.config["UPLOAD_FOLDER"] = 'tests/flask_app/test_data'
+   with app.test_client() as client:
+      yield client
+
+def test_get_upload_page(client):
+   response = client.get('/data')
+   assert response.status_code == 200
```

Comment:
Is 200 supposed to mean that everything ran properly?

Reply:
Yes, generally 200 means "OK". We could be more descriptive with our return codes in the future (here is a good summary of http return codes: https://restfulapi.net/http-status-codes/).